docusaurus/docs/engineering/knowledge-base/development-lifecycle.md

```diff
@@ -4,154 +4,230 @@ sidebar_position: 7
 
 # Development Lifecycle
```

Comment:
Not for this PR, but I think this page is long and we should consider making it a folder with multiple sections. It should also be moved out of engineering and in the volunteering section.